src/interp_routines.F

```diff
@@ -440,14 +440,10 @@ subroutine vinterp_weno5(stag,ix,jy,kz,c1,c2,rrw,dumz,a,pdef,pdefweno,weps)
 !  w-staggered variable:
 
   ELSEIF( stag.eq.4 )THEN
-#ifdef _OPENACC
-    stop 'vinterp_weno5:  stag == 4 not supported on OpenACC'
-#endif
 
-!$omp parallel do default(shared)   &
-!$omp private(i,j,k,wbar,cc1,cc2)
+    !$omp parallel do default(shared)  private(i,j,k,wbar)
+    !$acc parallel loop gang vector collapse(3) default(present)
```

Comment:
According to the PR description, adding lots of OpenACC directives below does not change the verification answers. Does it mean that these calculations are not called at all?

Reply:
@sjsprecious, none of the modified code in this PR has ever been executed using any of the existing namelists that we have used.